web_responsive/static/src/css/web_responsive.scss

```
@@ -421,20 +425,21 @@ html .o_web_client .o_main .o_main_content {
                     min-width: 30%;
                     overflow: auto;
 
-                    .o_chatter_topbar,
-                    .o_thread_composer {
-                        position: sticky;
-                    }
-
-                    .o_chatter_topbar {
+                    .o_chatter_header_container {
+                        padding-top: $grid-gutter-width*0.5;
                         top: 0;
-                        z-index: 2;
+                        position: sticky;
                         background-color: $o-view-background-color;
-                    }
-
-                    .o_thread_composer {
-                        top: $o-statusbar-height;
                         z-index: 1;
+
+                        .o_chatter_topbar {
+                            margin-top: 0;
+                        }
+
+                        .o_attachments_previews {
+                            overflow: auto;
+                            max-height: $o-mail-attachment-image-size * 2 * 3;
```

:thinking: Why not just this?

```suggestion
                            max-height: $o-mail-attachment-image-size * 6;
```

---

max-height = SIZE * ROWS
SIZE = $o-mail-attachment-image-size * 2
ROWS = 3

I don't know what is "best"... simple for humans (adding a comment to explain the line) or simple for computers (commit your changes) xD

---

Answering to the previous version of your answer, best is always to make it easier for humans as long as it doesn't affect performance.

I'd appreciate a comment then explaining this. Not a blocker though. :blush: